src/Addon/Plugin/PluginCriteria.php

```diff
@@ -218,13 +233,15 @@ public function getCacheKey()
     /**
      * Get the cache prefix.
      *
-     * @return null|string
+     * @return string
      */
     public function getCachePrefix()
     {
-        if ($model = $this->getModel()) {
```

Comment:
Why only include that when there is a model? If it is explicitly set, why not use it?